src/containers/SearchPage/SearchPageWithGrid.js

```diff
@@ -135,12 +136,17 @@ export class SearchPageComponent extends Component {
         // We should always trust urlQueryParams with those.
         // The same applies to keywords, if the main search type is keyword search.
         const keywordsMaybe = isMainSearchTypeKeywords(config) ? { keywords } : {};
+        const datesAndSeatsMaybe = getDatesAndSeatsMaybe(
+          prevState.currentQueryParams,
```

Comment:
I'm not entirely sure if this should be `mergedQueryParams` instead of `prevState.currentQueryParams` 
(aka I can't remember if some liveEdit or page load scenario has empty prevState and therefore the "old" values would come entirely from urlQueryParams).

Reply:
I tested this and changed it to use mergedQueryParams for clarity 👍